projenrc/codecov.ts

```diff
@@ -24,6 +24,7 @@ export class CodeCovWorkflow extends Component {
     });
 
     this.workflow.addJob('collect', {
+      runsOn: ['ubuntu-latest'],
```

Comment:
Use this like everywhere else?
```suggestion
      runsOn: ['aws-cdk_ubuntu-latest_4-core'],
```

Reply:
sure, but quite a few of our workflows are using ubuntu-latest.